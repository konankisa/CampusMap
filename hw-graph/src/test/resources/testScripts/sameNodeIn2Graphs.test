# Creates 2 empty graphs, adds the same node to both,
# and lists the node

# Create a graph
CreateGraph graph1

# Create another graph
CreateGraph graph2

# Add a node to graph1
AddNode graph1 node

# Add a node to graph2
AddNode graph2 node

# List the node in graph1
ListNodes graph1

# List the node in graph2
ListNodes graph2