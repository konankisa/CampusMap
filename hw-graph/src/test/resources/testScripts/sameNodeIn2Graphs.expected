# Creates 2 empty graphs, adds the same node to both,
# and lists the node

# Create a graph
created graph graph1

# Create another graph
created graph graph2

# Add a node to graph1
added node node to graph1

# Add a node to graph2
added node node to graph2

# List the node in graph1
graph1 contains: node

# List the node in graph2
graph2 contains: node